django_react_proj/users-fe/src/components/TaskButtonModal/TaskButtonModal.module.css

```diff
@@ -23,3 +23,8 @@
     min-width: 20px;
     border-radius: 4px;
 }
+.inbtn{
```

Comment:
You can move it to some e.g. `common.css` and reuse it in those components 

`import commonStyles from '[..]/common.css`